test/widgets/action_sheet_test.dart

```diff
@@ -227,6 +227,7 @@ void main() {
       void checkButtons() {
         check(actionSheetFinder).findsOne();
         checkButton('Mark channel as read');
+        checkButton('List of topics');
```

Comment:
nit: match the order these appear in the action sheet (and in the code, and the tests below)
```suggestion
        checkButton('List of topics');
        checkButton('Mark channel as read');
```